Integrations/AWS-EC2/AWS-EC2.py

```diff
@@ -0,0 +1,2678 @@
+        })
+        sts_response = sts_client.assume_role(**kwargs)
+        client = boto3.client(
+            service_name=service,
+            region_name=AWS_DEFAULT_REGION,
+            aws_access_key_id=sts_response['Credentials']['AccessKeyId'],
+            aws_secret_access_key=sts_response['Credentials']['SecretAccessKey'],
+            aws_session_token=sts_response['Credentials']['SessionToken'],
+            verify=VERIFY_CERTIFICATE,
+            config=config
+        )
+    else:
+        if region is not None:
+            client = boto3.client(
+                service_name=service,
+                region_name=region,
+                aws_access_key_id=AWS_access_key_id,
+                aws_secret_access_key=AWS_secret_access_key,
+                verify=VERIFY_CERTIFICATE,
+                config=config
+            )
+        else:
+            client = boto3.client(
+                service_name=service,
+                region_name=AWS_DEFAULT_REGION,
+                aws_access_key_id=AWS_access_key_id,
+                aws_secret_access_key=AWS_secret_access_key,
+                verify=VERIFY_CERTIFICATE,
+                config=config
+            )
+
+    return client
+
+
+def parse_filter_field(filter_str):
+    filters = []
+    regex = re.compile(r'name=([\w\d_:.-]+),values=([ /\w\d@_,.*-]+)', flags=re.I)
+    for f in filter_str.split(';'):
+        match = regex.match(f)
+        if match is None:
+            demisto.log('could not parse filter: %s' % (f,))
+            continue
+
+        filters.append({
+            'Name': match.group(1),
+            'Values': match.group(2).split(',')
+        })
+
+    return filters
+
+
+def parse_tag_field(tags_str):
+    tags = []
+    regex = re.compile(r'key=([\w\d_:.-]+),value=([ /\w\d@_,.*-]+)', flags=re.I)
+    for f in tags_str.split(';'):
+        match = regex.match(f)
+        if match is None:
+            demisto.log('could not parse field: %s' % (f,))
+            continue
+
+        tags.append({
+            'Key': match.group(1),
+            'Value': match.group(2)
+        })
+
+    return tags
+
+
+class DatetimeEncoder(json.JSONEncoder):
+    # pylint: disable=method-hidden
+    def default(self, obj):
+        if isinstance(obj, datetime):
+            return obj.strftime('%Y-%m-%dT%H:%M:%S')
+        elif isinstance(obj, date):
+            return obj.strftime('%Y-%m-%d')
+        # Let the base class default method raise the TypeError
+        return json.JSONEncoder.default(self, obj)
+
+
+def parse_resource_ids(resource_id):
+    id_list = resource_id.replace(" ", "")
+    resourceIds = id_list.split(",")
+    return resourceIds
+
+
+def multi_split(data):
+    data = data.replace(" ", "")
+    data = data.split(";")
+    return data
+
+
+def parse_date(dt):
+    arr = dt.split("-")
+    return (datetime(int(arr[0]), int(arr[1]), int(arr[2]))).isoformat()
+
+
+"""MAIN FUNCTIONS"""
+
+
+def describe_regions(args):
```

Comment:
this I a general comment on all of the command functions. it is not in the code convention.
command function names should end in `command` and there is no reason to pass args to them. they should access demisto args in the beginning of the function.